src/org/omegat/gui/theme/DefaultFlatTheme.java

```diff
@@ -0,0 +1,272 @@
+import javax.swing.UIDefaults;
+import javax.swing.UIManager;
+import javax.swing.border.CompoundBorder;
+import javax.swing.border.EmptyBorder;
+import javax.swing.border.MatteBorder;
+
+import org.omegat.util.OStrings;
+import org.omegat.util.Platform;
+import org.omegat.util.gui.ResourcesUtil;
+import org.omegat.util.gui.RoundedCornerBorder;
+
+
+public class DefaultFlatTheme extends DelegatingLookAndFeel {
+
+    private static final String NAME = OStrings.getString("THEME_OMEGAT_DEFAULT_NAME");
+
+    public static void loadPlugins() {
+        UIManager.installLookAndFeel(NAME, DefaultFlatTheme.class.getName());
+    }
+
+    public static void unloadPlugins() {
+    }
+
+    public DefaultFlatTheme() throws Exception {
+        super();
+    }
+
+    @Override
+    public String getName() {
+        return NAME;
+    }
+
+    @Override
+    public String getID() {
+        return "OmegaT"; // NOI18N
+    }
+
+    @Override
+    public String getDescription() {
+        return OStrings.getString("THEME_OMEGAT_DEFAULT_DESCRIPTION");
+    }
+
+    @Override
+    public UIDefaults getDefaults() {
+        // Use system LAF ID because we use the ID to see if it's e.g. Windows
+        return setDefaults(systemLookAndFeel.getDefaults(), systemLookAndFeel.getID());
+    }
+
+    /**
+     * Apply theme default values to the supplied {@link UIDefaults} object.
+     * Suitable for third-party themes that want to modify the default theme.
+     *
+     * @param defaults the {@link UIDefaults} object such as from
+     *                 {@link LookAndFeel#getDefaults()}
+     * @param lafId    the ID of the LAF that will make use of the defaults (from
+     *                 {@link LookAndFeel#getID()})
+     * @return the modified {@link UIDefaults} object
+     */
+    public static UIDefaults setDefaults(UIDefaults defaults, String lafId) {
+        // Colors
+        // #EEEEEE on Metal & OS X LAF
+        Color standardBgColor = defaults.getColor("Panel.background");
+        // #EEEEEE -> #F6F6F6; Lighter than standard background
+        Color activeTitleBgColor = adjustRGB(standardBgColor, 0xF6 - 0xEE);
+        // #EEEEEE -> #DEDEDE; Darkest background
+        Color bottomAreaBgColor = adjustRGB(standardBgColor, 0xDE - 0xEE);
+        // #EEEEEE -> #9B9B9B; Standard border. Darker than standard background.
+        Color borderColor = adjustRGB(standardBgColor, 0x9B - 0xEE);
+        defaults.put("OmegaTBorder.color", borderColor);
+        // #EEEEEE -> #575757; Darkest border
+        Color statusAreaColor = adjustRGB(standardBgColor, 0x57 - 0xEE);
+
+        // General highlight & shadow used in a lot of places
+        //
+        // FIXME: VLDocking values have to be set to the "developer defaults"
+        // not the "LAF defaults" because that's where
+        // DockingUISettings#installUI puts them
+        UIManager.put("VLDocking.highlight", activeTitleBgColor);
+        UIManager.put("VLDocking.shadow", statusAreaColor);
```

Comment:
question: Are there any reason `VLDocking*` parameters should set through `UIManager.put`, not set `defaults`?

Reply:
Yes. I mentioned it in the comment right above here, but we should probably document it better elsewhere too.

`UIManager` manages three "layers" of "defaults" (= sets of key-value settings). From [the Javadoc](https://docs.oracle.com/javase/7/docs/api/javax/swing/UIManager.html):

> 1. Developer defaults. With few exceptions Swing does not alter the developer defaults; these are intended to be modified and used by the developer.
> 2. Look and feel defaults. The look and feel defaults are supplied by the look and feel at the time it is installed as the current look and feel (`setLookAndFeel()` is invoked). The look and feel defaults can be obtained using the `getLookAndFeelDefaults()` method.
> 3. Sytem defaults. The system defaults are provided by Swing.

In our `setDefaults` implementation we can only set values on layer 2 (LAF defaults). However VLDocking's [`installUI`](https://github.com/omegat-org/vldocking/blob/1a786b686b4e94bc8a63e477a592ea4e7ee2c71b/src/main/java/com/vlsolutions/swing/docking/ui/DockingUISettings.java#L626) puts all of its settings into layer 1.

We can't override the VLDocking defaults from layer 2, so we have no choice but to put them in layer 1 via `UIManager#put*`.

I think the correct solution is to change VLDocking: it should let you install its defaults into layer 2. But in the meantime we can work around it by using `UIManager#put*` in our `setDefaults` method.

Reply:
Got it.

 I'd like to move a sample plugin to be a `theme-skeleton` and encourage other users to propose better themes.
That is because, I've put the description on the plugin project's README.